android/app/build.gradle

```diff
@@ -45,7 +45,7 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.unicornsonlsd.finamp"
-        minSdkVersion 21
+        minSdkVersion 26
```

Comment:
Is changing the minimum API level required here? I'd rather not drop so many (albeit old) Android versions

Reply:
I tried building the app with the old version and it threw an error stating that the minimum API version is 26. 🤔

Reply:
> Next, in res/mipmap-anydpi-v26/ic_launcher.xml, create alternative drawable resources in your app for backward compatibility with Android 8.0 (API level 26). Use the <adaptive-icon> element to define the foreground, background, and monochromatic layer drawables for your icons. The <foreground>, <background>, and <monochrome> inner elements support the android:drawable attribute.

This was on the site that I provided in the MR description, seems like anything lower than Android 8.0 (API level 26) is no longer supported with this change.